okta/sdk_test.go

```diff
@@ -76,6 +79,21 @@ func testAuthHeader(t *testing.T, r *http.Request) {
 	}
 }
 
+func testBody(t *testing.T, r *http.Request, want interface{}) {
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		t.Errorf("Error reading request body: %v", err)
+	}
+	// TODO: we check if newline is in body but not if it is the last character
+	if bytes.Contains(body, []byte("\n")) {
+		body = body[:len(body)-1]
```

Comment:
admittedly, i'm not completely happy with this function and will likely come back to it to improve.

Reply:
Perhaps a regex replace would be better in this case?

Reply:
i think for a regex replace, the var needs to be a string? i'll look into this. i am interested in using bytes.Equal below instead of a string comparison to ideally handle comparing multilevel json.

Reply:
Looks like https://golang.org/pkg/regexp/#Regexp.ReplaceAll takes a byte and returns a byte